Add March update (#84)
ThePHPF/thephp.foundation: source/_posts/2023-03-31-php-foundation-update-march-2023.md

* Every month we provide developer plans – a list of tasks that the PHP Foundation developers will work on in the near future. Advisory Board members can discuss these plans among their teams and provide us with feedback that they feel we should consider.
* Advisory board members are welcome to share their pain points or ideas on behalf of their communities or organizations. Suggestions may be discussed by other members and will be considered by the foundation developers.

> ⚠️ Note: All language decisions go through the established Request for Comments (RFC) process, which is not affiliated to the PHP Foundation, and the language decisions are the matter of the PHP Internals community. It means that some of the PHP Foundations efforts might not make it to the language. But with the Advisory board discussions, we expect increased quality and acceptance rate of the foundation's RFCs.

Reviewer: ```suggestion
⚠️ Note: All language decisions go through the established Request for Comments (RFC) process, which is not affiliated with the PHP Foundation. Language decisions are purview of PHP Internals. It means that some of the PHP Foundation’s efforts may not make it into the language. But with the Advisory Board discussions, we expect increased quality and acceptance rate of the foundation’s RFCs.
```
